src/query/expression/src/kernels/scatter.rs

```diff
@@ -121,6 +123,18 @@ impl Column {
                 indices,
                 scatter_size,
             ),
+            Column::Date(column) => Self::scatter_scalars::<Int32Type, _>(
```

Comment:
If we map DateType into Int32Type, how could we register functions for DateType ?